### PR TITLE
fix: use module options instead of env variables

### PR DESCRIPTION
**Pull request by kubanedo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,18 +34,32 @@ yarn add nuxt-website-lock # or npm install nuxt-website-lock
...
 
-* `These variables need to be passed via .env or env key in nuxt.config.js`
+* `These variables need to be passed via module options in nuxt.config.js`
 
-| Env variable name        | Default value   | Possile values                            |
+| Option name              | Default value   | Possible values                           |
 |--------------------------|-----------------|-------------------------------------------|
-| `WEB_SITE_LOCK_ENABLED`  | default: `null` | options: `true`/`false`                   |
-| `WEB_SITE_LOCK_PASSWORD` | default: `null` | any string (will be simply hashed to md5) |
+| `web_site_lock_enabled`  | default: `null` | options: `true`/`false`                   |
```
Comment (truesteps) [inline README.md]: please let's use more conventional naming, so go with enabled for this option

File: README.md [hunk 2]
```diff
@@ -34,18 +34,32 @@ yarn add nuxt-website-lock # or npm install nuxt-website-lock
...
+| `web_site_lock_password` | default: `null` | any string (will be simply hashed to md5) |
```
Comment (truesteps) [inline README.md]: here i would go with the password option name, keep in mind in previous options we use camelCase and then suddenly switch to snake case in this instance so please change this to keep it consistent

File: README.md [hunk 3]
```diff
@@ -34,18 +34,32 @@ yarn add nuxt-website-lock # or npm install nuxt-website-lock
...
-After setting up the `.env`, all you need to do is apply the `WebsiteLock` route middleware to the page(s), which should be locked
+```js
+{
+  modules: [
+    // Simple usage
+    'nuxt-website-lock',
+
+    // website-lock has to be before axios so axios instance is available at websitelock module
+    // to enable validation of token on BE change "enableServerValidation" to true
+    ['nuxt-website-lock', {enableServerValidation: false, web_site_lock_enabled: true, web_site_lock_password: 'pass'}],
```
Comment (truesteps) [inline README.md]: ['nuxt-website-lock', {enableServerValidation: false, enabled: true, password: 'pass'}],

File: README.md [hunk 4]
```diff
@@ -34,18 +34,32 @@ yarn add nuxt-website-lock # or npm install nuxt-website-lock
...
 
 ## Usage
-To enable the website lock, you need to pass the app 2 `.env` variables
+To enable the website lock, you need to pass the app 2 `module options` variables
 
-* `These variables need to be passed via .env or env key in nuxt.config.js`
+* `These variables need to be passed via module options in nuxt.config.js`
 
-| Env variable name        | Default value   | Possile values                            |
+| Option name              | Default value   | Possible values                           |
```
Comment (truesteps) [inline README.md:41]: plaese include the enableServerValidation option too, with its default value being false.

Thanks!

File: example/nuxt.config.js [hunk 5]
```diff
@@ -6,12 +6,17 @@ module.exports = {
...
-    WEB_SITE_LOCK_PASSWORD: 'superSecretPassword'
+    NUXT_API_URL: 'http://localhost/bistro/tady-hub'
   },
   modules: [
-    { handler: require('../'), options: { enableServerValidation: false } },
+    {
+      handler: require('../'),
+      options: {
+        enableServerValidation: false,
+        web_site_lock_password: 'superSecretPassword',
```
Comment (truesteps) [inline example/nuxt.config.js]: password

File: example/nuxt.config.js [hunk 6]
```diff
@@ -6,12 +6,17 @@ module.exports = {
...
+        web_site_lock_enabled: true
```
Comment (truesteps) [inline example/nuxt.config.js]: enabled

File: lib/main.js [hunk 7]
```diff
@@ -12,7 +12,7 @@ export const getWebsiteLock = ({ options, ctx, storage }) => {
         } else {
           const hashedToken = md5(token)
 
-          if (md5(process.env.WEB_SITE_LOCK_PASSWORD) === hashedToken) {
+          if (md5(options.web_site_lock_password) === hashedToken) {
```
Comment (truesteps) [inline lib/main.js]: options.password

File: lib/main.js [hunk 8]
```diff
@@ -35,7 +35,7 @@ export const getWebsiteLock = ({ options, ctx, storage }) => {
     },
 
     async checkIfUserCanRedirect () {
-      if (!JSON.parse(process.env.WEB_SITE_LOCK_ENABLED)) {
+      if (!JSON.parse(options.web_site_lock_enabled)) {
```
Comment (truesteps) [inline lib/main.js]: options.enabled

File: lib/main.js [hunk 9]
```diff
@@ -56,7 +56,7 @@ export const getWebsiteLock = ({ options, ctx, storage }) => {
             // do nothing
           }
         } else {
-          isAuthenticated = md5(process.env.WEB_SITE_LOCK_PASSWORD) === token
+          isAuthenticated = md5(options.web_site_lock_password) === token
```
Comment (truesteps) [inline lib/main.js]: options.password

**Review by truesteps — CHANGES_REQUESTED** [review]
CR done, please perform requested changes and then I'll merge it :)

**Comment by truesteps** [comment]
@kubanedo one more thing, please delete the table and the whole usage section, keep the module options and move the options into the module options table :) thanks

**Comment by kubanedo** [comment]
@truesteps upraveno :)

**Comment by krftcz** [comment]
nice work!

**Pull request merged**